include/openspace/rendering/renderengine.h

```diff
@@ -68,6 +68,8 @@ class RenderEngine : public properties::PropertyOwner {
     RenderEngine();
     virtual ~RenderEngine() override;
 
+    FramebufferRenderer* renderer();
```

Comment:
With all of the functions above being `const`, maybe this can be:
`const FramebufferRenderer* renderer() const`?

or even
`const FramebufferRenderer& renderer() const`?